### DISCUSSION TITLE
go2rtc not starting stream

### DISCUSSION THREAD

**Discussion by maddavo** [gtxaspec/wz_mini_hacks#685] [Q&A]
I have a Wyze Cam v3 and I want to get a rtsp stream working so I can use it with Home Assistant.  I have tried to follow the steps in the wiki and read further info here and in the go2rtc repository, but I seem to have reached an impasse.  I just cannot get a stream going.  I need a clever person to give me advice.

The camera firmware is 4.36.11.7095 .
I can see the camera view with the Wyze app.
I have installed mini hacks OK.
I have managed to ssh into the camera.
I have edited the wz_mini configuration file setting all the go2rtc booleans to true.
I have rebooted the camera.
I can get to the go2rtc gui in a browser.

The gui page looks like this:
![Desktop Screenshot 2023 12 12 - 23 41 15 76](https://github.com/gtxaspec/wz_mini_hacks/assets/1432875/6587a140-a7de-4b5f-9730-388db3c962c2)

I think this indicates there is something amiss since it shows "0" streams right?

Opening a browser to http://(camera ip):1984/stream.html?src=1080p throws an error "mse: streams: exec: exit status 1 error" and no stream

I have not edited the go2rtc.yml , it just has the three default exec:ffmpeg lines for each stream, /dev/video1 for 1080p and /dev/video2 for 360p, which I think is correct for a Wyze Cam v3.

What am I missing?  What is the trick to get this going?

It feels so close, surely I'm just missing some small step somewhere.  Thanks in advance!

**Comment by archandanime** [comment]
go2rtc on wz_mini_hacks doesn't work for any firmware version beyond `4.36.9.139` as stated [here](https://github.com/gtxaspec/wz_mini_hacks/wiki/Firmware-Support).
But you can rollback using older firmware file using the [official tutorial](https://support.wyze.com/hc/en-us/articles/360031490871-How-to-flash-your-Wyze-Cam-firmware-manually):
[demo_wcv3_4.36.9.139.zip](https://github.com/gtxaspec/wz_mini_hacks/files/13649768/demo_wcv3_4.36.9.139.zip) (SHA256: `01f3fcf6ab3e8670f09f2384f24916a63588555bd43d21be68aa2cd8cf43c364`)

**Comment by Bob-AOL** [comment]
Or if all that is desired is a basic RTSP stream from the camera, the Wyze Beta v4.61.0.1/3 is a quick and easy solution that works fine with or without wz_mini. When used with wz_mini it will be the dominant RTSP server. I use it that way both wired and wireless with no issues.

**Comment by maddavo** [comment]
> go2rtc on wz_mini_hacks doesn't work for any firmware version beyond `4.36.9.139` as stated [here](https://github.com/gtxaspec/wz_mini_hacks/wiki/Firmware-Support). But you can rollback using older firmware file using the [official tutorial](https://support.wyze.com/hc/en-us/articles/360031490871-How-to-flash-your-Wyze-Cam-firmware-manually): [demo_wcv3_4.36.9.139.zip](https://github.com/gtxaspec/wz_mini_hacks/files/13649768/demo_wcv3_4.36.9.139.zip) (SHA256: `01f3fcf6ab3e8670f09f2384f24916a63588555bd43d21be68aa2cd8cf43c364`)

Ah yes, I did note that table, however my understanding was muddied somewhat by the **Note [1]** which states that "rtsp streaming... etc. _will work_".  I took the preceding "No libcallback functions" to be a phrase unto it's own, giving reason as to why the examples "will work".  To avoid further confusion by limited individuals such as myself or others perhaps unfamiliar with the nature of libcallback functions encompassing those examples, it may be helpful if the note were reworded to say "No libcallback functions will work" and then list the examples.

Thank you greatly for your assistance.  I had suspected the firmware may be the issue and went searching on the wyze website for the old firmware but was unsuccessful in locating a download link.

**Reply by yuuzhan** [reply]
I have a similar issue running 4.49.1.47 on a Wyze Pan v2 (which I think implies it should be working?)

I pasted the diagnostic logs from the webui here - but let me know if you want me to post another error log, or need any other data?

https://pastebin.com/6jZbRs3B
